### PR TITLE
Mongo 2.6.12 ami

### PR DESCRIPTION
**Pull request by jennysivapalan**
Added an AMI with 2.6.12 Mongo.

I've also created a script to help initialise the database and user credentials. This relies on a file being passed which must contain the following properties. 

```
mongo.database.name=
mongo.database.username=
mongo.database.password=
```

cc @philmcmahon @sihil

### CODE REVIEW

File: packer/resources/features/mongo26/install.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+# Install MongoDB 2.6
+
+SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
+
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' > /etc/apt/sources.list.d/mongodb.list
```
Comment (philmcmahon) [inline packer/resources/features/mongo26/install.sh:9]: Did we need to update this to org rather than 10gen?

**Comment by philmcmahon** [comment]
👍

File: packer/resources/features/mongo26/install.sh (hunk 1 above)
Comment (jennysivapalan) [inline packer/resources/features/mongo26/install.sh:9]: No the folder is still `10gen` but the .deb has changed to`org`

Comment (philmcmahon) [inline packer/resources/features/mongo26/install.sh:9]: ok great!

**Pull request merged**